default/web_tt2/latest_lists.tt2

```diff
@@ -1,45 +1,46 @@
 <!-- latest_lists.tt2 -->
 
 <h2>
- [% IF count %]
-   [%|loc(count)%]The %1 newest lists[%END%]
- [% ELSE %]
-   [%|loc%]New lists[%END%]
- [% END %]
-
- [% IF for %]
-   [%|loc(for)%] for %1 days [%END%]
- [% END %]
+    [% IF count %]
+        [%|loc(count)%]The %1 newest lists[%END%]
+    [% ELSE %]
+        [%|loc%]New lists[%END%]
+    [% END %]
+
+    [% IF for %]
+        [%|loc(for)%] for %1 days [%END%]
+    [% END %]
 </h2>
 
 <br />
 
 <table class="responsive table_style" >
-<caption>[%|loc%] The latest lists of this robot[%END%] </caption>
-<tr class="color_light">
-   <th>[%|loc%]List name[%END%]</th>
-   <th>[%|loc%]Creation date[%END%]</th>
-   <th>[%|loc%]Subject[%END%]</th>
- </tr>
-
-[% FOREACH l = latest_lists %]
-
-[% IF dark == '1' %]
-  <tr valign="top">[% SET dark = 0 %]
-[% ELSE %]
-  <tr class="color0">[% SET dark = 1 %]
-[% END %]
-   <td>
-     <a href="[% 'info' | url_rel([l.name]) %]" ><strong>[%|obfuscate(conf.spam_protection) %][% l.name %]@[% domain %][% END %]</strong></a>
-   </td>
-
-  <td> [% l.date %] </td>
-  <td> [% l.subject %] </td>
- </tr>
-
- [% END %]
-
- <br />
+    <caption>[%|loc%] The latest lists of this robot[%END%] </caption>
+    <tr class="color_light">
+        <th>[%|loc%]List name[%END%]</th>
+        <th>[%|loc%]Creation date[%END%]</th>
+        <th>[%|loc%]Subject[%END%]</th>
+    </tr>
+
+    [% FOREACH l = latest_lists %]
+        [% IF dark == '1' %]
+            <tr valign="top">
+            [% SET dark = 0 %]
+        [% ELSE %]
+            <tr class="color0">[% SET dark = 1 %]
+        [% END %]
+                <td>
+                    <a href="[% 'info' | url_rel([l.name]) %]" >
+                        <strong>
+                            [%|obfuscate(conf.spam_protection) %][% l.name %]@[% domain %][% END %]
+                        </strong>
+                    </a>
+                </td>
+                <td> [% l.date %] </td>
+                <td> [% l.subject %] </td>
+            </tr>
+    [% END %]
 </table>
+<br />
```

Comment:
I think this `<br />` is not required at the beginning.

Reply:
Fixed in https://github.com/sympa-community/sympa/pull/582/commits/7b3a39c6ac7c013a0990a7033f81118d1943e2de

Reply:
Confirmed.